Vungle/src/main/java/com/mopub/mobileads/VungleInterstitial.java

```diff
@@ -288,69 +282,88 @@ public void run() {
         }
 
         @Override
-        public void onUnableToPlayAd(@NonNull String placementReferenceId, String reason) {
-            if (mPlacementId.equals(placementReferenceId)) {
-                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onUnableToPlayAd - Placement ID: " +
-                        placementReferenceId + ", reason: " + reason);
+        public void onAdPlayError(@NonNull String placementId, VungleException exception) {
+            if (mPlacementId.equals(placementId)) {
+                MoPubLog.log(getAdNetworkId(), CUSTOM, ADAPTER_NAME, "onAdPlayError - Placement ID: " +
+                        placementId + ", reason: " + exception.getLocalizedMessage());
+
                 mIsPlaying = false;
+                final MoPubErrorCode errorCode = mapErrorCode(exception.getExceptionCode());
 
                 mHandler.post(new Runnable() {
                     @Override
                     public void run() {
-                        if (mInteractionListener == null && mLoadListener != null) {
-                            mLoadListener.onAdLoadFailed(MoPubErrorCode.NETWORK_NO_FILL);
-                            MoPubLog.log(getAdNetworkId(), LOAD_FAILED, ADAPTER_NAME,
-                                    MoPubErrorCode.NETWORK_NO_FILL.getIntCode(),
-                                    MoPubErrorCode.NETWORK_NO_FILL);
-                        } else if (mInteractionListener != null) {
-                            mInteractionListener.onAdFailed(MoPubErrorCode.NETWORK_NO_FILL);
-                            MoPubLog.log(getAdNetworkId(), SHOW_FAILED, ADAPTER_NAME,
-                                    MoPubErrorCode.NETWORK_NO_FILL.getIntCode(),
-                                    MoPubErrorCode.NETWORK_NO_FILL);
+                        if (mInteractionListener != null) {
```

Comment:
The same issue about mLoadListener logic: https://github.com/mopub/mopub-android-mediation/commit/83d908ba389cc6c2f931c416b2341cdb00d57bc7#diff-c14a90e6291945340ba12d6896ee4aebe86e46a9ee70ecc31a757d287da8e41b